### PR TITLE
Hol oke based tutorial

### PR DESCRIPTION
**Pull request by nagypeter**
WebLogic Kubernetes Operator Hands-On-Lab running on OKE (Oracle Kubernetes Engine)

Doesn't require desktop just a browser because it uses OCI Cloud Shell.
Demonstrates major features of WebLogic Kubernetes Operator.

### CODE REVIEW

File: kubernetes/hands-on-lab/tutorials/node.selector.license.ocishell.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Oracle WebLogic Server Kubernetes Operator Tutorial #
 
-### Assign WebLogic pods to licensed nodes ###
+### Assign WebLogic domain to selected nodes ###
```
Comment (rosemarymarano) [inline kubernetes/hands-on-lab/tutorials/node.selector.license.ocishell.md]: Assign WebLogic domain -> Assign a WebLogic domain

File: kubernetes/hands-on-lab/tutorials/node.selector.license.ocishell.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 This use case is similar to the [Assign WebLogic Pods to Nodes](node.selector.ocishell.md) lab, where individual servers/pods were assigned to specific node(s). However, the focus in this use case on the license coverage.
 
-At v1.13, Kubernetes supports clusters with up to 5000(!) nodes. However, certain software, such as WebLogic Server, requires a license. Using the `nodeSelector` feature, Kubernetes ensures that WebLogic pods end up on licensed worker node(s).
+Starting from v1.13, Kubernetes supports clusters with up to 5000(!) nodes. However, in certain situation you need to deploy WebLogic domain on selected node(s). Using the `nodeSelector` feature, Kubernetes ensures that all WebLogic pods belong to the domain end up on licensed worker node(s).
```
Comment (rosemarymarano) [inline kubernetes/hands-on-lab/tutorials/node.selector.license.ocishell.md]: Starting from v1.13 -> Starting at v1.13 
in certain situation -> in certain situations
deploy WebLogic domain -> deploy a WebLogic domain
all WebLogic pods belong -> all WebLogic pods that belong

File: kubernetes/hands-on-lab/domain.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (mriccell) [inline kubernetes/hands-on-lab/domain.yaml:1]: You need to fix the Copyright
Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.

Comment (nagypeter) [reply kubernetes/hands-on-lab/domain.yaml]: I changed the Copyright date

File: kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+If you want to create a new branch to retain commits you create, you may
+do so (now or later) by using -b with the checkout command again. Example:
+
+  git checkout -b new_branch_name
+
+Checking out files: 100% (8396/8396), done.
+```
+#### Prepare the environment ####
+In order to use Helm to install and manage the operator, you need to ensure that the service account has the `cluster-admin` role.  The default is `default` in namespace `kube-system`.  You can give that service
```
Comment (rjeberhard) [inline kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md:35]: This is only necessary for Helm 2.x.  I'm going to run through this workshop and make sure all is correct for Helm 3.

**Pull request merged**